Makefile

```diff
@@ -16,17 +16,22 @@ help:
 	@echo ""
 	@echo "  Targets"
 	@echo ""
-	@echo "    deps       Install python packages"
-	@echo "    install    Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
-	@echo "    uninstall  Uninstall scripts and $(SHAREDIR)"
-	@echo "    docker     Build Docker image"
+	@echo "    deps-ubuntu	Install system packages (on Debian/Ubuntu)"
+	@echo "    deps       	Install python packages"
+	@echo "    install    	Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
+	@echo "    uninstall  	Uninstall scripts and $(SHAREDIR)"
+	@echo "    docker     	Build Docker image"
 	@echo ""
 	@echo "  Variables"
 	@echo ""
 	@echo "    PREFIX  Directory to install to ('$(PREFIX)')"
 
 # END-EVAL
 
+# Install system packages (on Debian/Ubuntu)
+deps-ubuntu:
+	apt-get install -y make python3 python3-pip python3-venv openjdk-8-jre-headless ghostscript
```

Comment:
```suggestion
	apt-get install -y default-jre-headless python3 python3-venv ghostscript
```

Reply:
`openjdk-8-jre-headless` would break the installation on any current Ubuntu or Debian distribution. The other removed entries are redundant.

Reply:
As soon as there are (hopefully positive) experiences with `openjdk-11-jre-headless`, we should also update the README.

Reply:
The problem is that PRImA's PAGE libraries **don't work** with Java 11 (yet). If this interferes with other components on your local host installation, consider using the Docker option or making parallel/post-installation configs (`update-java-alternatives`). We could also try to work our way to pointing directly to the Java runtime, but this may be more fragile in general.

Reply:
Then the build should neverless handle that case and skip parts which don't work because of missing packages - ideally with a informational message.

Reply:
> Then the build should neverless handle that case and skip parts which don't work because of missing packages - ideally with a informational message.

I don't understand. Our target system is still Ubuntu 18.04 – we have to stick with **some** reference point, at least for our Docker builds. There is no problem with `openjdk-8-jre-headless` in that version. If you are concerned with newer Ubuntu versions, why not install some Java 8 PPA yourself when you see the error, or use the Docker image?

Reply:
I simply want that builds on newer Linux distributions work, too. Maybe I am lazy and don't want to think each time that this package is special and breaks `make deps-ubuntu`. Or I expect many issues from others who stumble into the same problem because they don't use an old Ubuntu LTS.

Reply:
Look, we've had this discussion before. `deps-ubuntu` is not about making installation automatic for all Debian/Ubuntu flavors. It is a pragmatic, stop-gap measure for all OCR-D modules to have a common reference point for Docker builds, nothing more. This repo is not special in any way, all others already use it that way. If you want automatic deployment on other distros, feel free to make new proposals for https://github.com/OCR-D/ocrd_all/issues/56 or https://github.com/OCR-D/spec/issues/131.

Reply:
Agreeing with @bertsky here, `deps-ubuntu` should, for the time being and in any case for Docker builds, be understood as `deps-ubuntu-18.04`.

Reply:
My suggested change does cover that case.

Reply:
> My suggested change does cover that case.

AFAICT it does not. `default-jre-headless` metadata on Ubuntu 18.04.4 and 18.04.2:
```
Depends: openjdk-11-jre-headless, java-common
```